react/w-books/src/app/screens/login/styles.module.scss

```diff
@@ -0,0 +1,73 @@
+
+@import '../../../scss/colors';
+
+$border-radius: 5px;
+$input-height: 30px;
+
+%button-base {
+  border: 3px solid $secondary-color;
+  border-radius: $border-radius;
+  height: $input-height;
+}
+
+%login-button-green {
+
+  @extend %button-base;
+  color: #FFF;
+  background-color: $secondary-color;
+
+  &:hover {
+    background-color: #FFF;
+    color: $secondary-color;
+  }
+}
+
+%login-button-white {
+
+  @extend %button-base;
+  color: $secondary-color;
+
+  &:hover {
+    color: #FFF;
+    background-color: $secondary-color;
+  }
+}
+
+.login-container {
+  background-color: #F5F5F5;
+  padding: 20px;
+  width: 300px;
+  border-top: 5px solid $primary-color;
+  margin: 20px auto;
+  display: flex;
+  flex-direction: column;
```

Comment:
Inside the scss folder you will see a file called **layout.scss**, there you can see general classes to make structure layouts, recommended is used that classes instead of calling these properties in your custom classes.